homeassistant/components/tplink/config_flow.py

```diff
@@ -110,11 +108,32 @@ def _update_config_if_entry_in_setup_error(
             != config.get(CONF_CONNECTION_TYPE)
         ):
             updates.pop(CONF_CREDENTIALS_HASH, None)
-        return self.async_update_reload_and_abort(
-            entry,
-            data=updates,
-            reason="already_configured",
-        )
+            if _LOGGER.isEnabledFor(logging.DEBUG):
```

Comment:
You can drop the `isEnabledFor` guard here since the argument are not expensive to compute here.
https://docs.python.org/3/howto/logging.html#optimization
> Formatting of message arguments is deferred until it cannot be avoided. However, computing the arguments passed to the logging method can also be expensive, and you may want to avoid doing it if the logger will just throw away your event. To decide what to do, you can call the [isEnabledFor()](https://docs.python.org/3/library/logging.html#logging.Logger.isEnabledFor) method which takes a level argument and returns true if the event would be created by the Logger for that level of call. You can write code like this:

Reply:
Ok that's done